repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py

```diff
@@ -22,7 +22,7 @@ def produce_error(self, error):
     def is_system_registered_and_attached(self):
         # TODO: put this to different actor and process it already during check
         # + phase
-        out = preparetransaction.call(['subscription-manager', 'list', '--consumed'], True)
+        out = call(['subscription-manager', 'list', '--consumed'], True)
```

Comment:
please use split=True for the split, so that a change in the signature won't break it